django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -147,7 +156,7 @@
     background-position: 0 -80px;
 }
 
-a.selector-chooseall, a.selector-clearall {
+button.selector-chooseall, button.selector-clearall {
```

Comment:
We need to apply some of these changes in `django/contrib/admin/static/admin/css/rtl.css` 
So that the arrows change direction when we are using a RTL language
![Uploading test_selectbox_selected_rows--selectbox-available-perms-none-selected--rtl.png…]()

Reply:
Also thank you for updating the button :rocket: